content/docs/apps/using-ssh.md

```diff
@@ -7,48 +7,10 @@ title: Using SSH
 
 Here's how to get a SSH shell in the environment where your app is running. You can use this to inspect how your app is operating, transfer files via SCP, or interact directly with your bound services. [More information about one-off tasks.]({{< relref "docs/getting-started/one-off-tasks.md" >}})
 
-{{% govcloud %}}
 ### CF SSH
```

Comment:
This is another one where we can probably consolidate this a bit more now that there's only one option, such as removing "Here’s how to get a SSH shell in the environment where your app is running." and the *CF SSH* subheading.

![screen shot 2017-04-24 at 4 25 49 pm](https://cloud.githubusercontent.com/assets/391313/25362741/b7ee6b54-290a-11e7-8b09-30c90dfef569.png)